docs/content/usage/windows-machine-images.md

```diff
@@ -0,0 +1,383 @@
+```
+
+Next, a Microsoft Windows install image must be copied to the
+`./resources/` directory.  The scripts have tested OK with images from
+`http://www.microsoft.com/`.  Currently, only Japanese images are
+supported. (English versions will be supported *very* soon.)
+
+The next step is to create a "build directory" that will hold the
+new seed image and various other files particular to one image build.
+We do this with the general purpose `build-dir-utils.sh` script.
+
+```bash
+./build-dir-utils.sh builddirs/manual-build-2008/ 0-init 2008
+```
+
+The directory `builddirs` should already exist as part of the Wakame-vdc
+repository.  The above command should create a new directory
+named `manual-build-2008`.  The third parameter (2008, in the example)
+is required and specifies which version of Windows will be installed.
+
+Next, specific settings for building the image are applied to the
+build directory.  A list of these can be found in the file
+`windows-image-build.ini`.  The default values should work, except for
+the ISO file name and the product key, which must be set.  This can be
+done by editing the file or by setting shell environment variables.
+
+```bash
+# the following was used in a recent test
+export ISO2008=7601.17514.101119-1850_x64fre_server_eval_ja-jp-GRMSXEVAL_JA_DVD.iso
+export KEY2008=none
+./build-dir-utils.sh builddirs/manual-build-2008/ 1-setup-install
+```
+
+In addition, the `1-setup-install` step checks to make sure all needed
+resources are in the `./resources/` directory.  If any are missing, an
+attempt will be made to download them.  The settings from
+`windows-image-build.ini` and environment variables that are gathered
+during this step will be used through the rest of the image
+installation process.
+
+### Partially Automatic Image Creation
+
+From this point, the image creation proceeds with multiple calls to
+the same `build-dir-utils.sh` script.  Always, the first argument is
+the build directory and the second argument is the name of the build
+step.  Because the build steps must be performed in order, the name of
+each step is prefixed with a number to indicate the ordering.
+
+Some of the build steps must be performed in the Windows user
+interface, and sometimes the best way to do these steps is manually.
+Therefore, some of the steps are just placeholders for actions that a
+user would do manually, and the script itself actually does not
+perform any action that affects the VM or Windows OS.  In this way,
+the steps themselves can serve as a concise documentation of *all* the
+steps necessary to build a Windows image.  The `build-dir-utils.sh`
+script both automates steps that are easy to automates, and helps the
+user not to forget those steps that are easier to perform manually.
+
+To make this easier, after the 0-init step, it is possible to go
+through the rest of the build issuing either one of the following
+commands:
+
+```bash
+./build-dir-utils.sh builddirs/manual-build-2008/ -do-next
```

Comment:
* After doing `-do-next` a few times I was told to use a vnc viewer to check if the 'Ctrl + Alt + Del' screen appears. We should mention at the top of the guide that a vnc viewer is required.

* When logging in with vnc, I noticed that the installation was stuck on product key entry. The guide did mention that product key needs to be set but it made it look like "none" would work as a value. If not, it should be more clear.